doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

```diff
@@ -49,6 +49,11 @@ To understand the following content better, you should understand the difference
   * `metadata` (Optional): See {ref}`Ray Jobs CLI API Reference <ray-job-submission-cli-ref>` for more details about the `--metadata-json` option.
   * `entrypointNumCpus` / `entrypointNumGpus` / `entrypointResources` (Optional): See {ref}`Ray Jobs CLI API Reference <ray-job-submission-cli-ref>` for more details.
   * `backoffLimit` (Optional, added in version 1.2.0): Specifies the number of retries before marking this RayJob failed. Each retry creates a new RayCluster. The default value is 0.
+  * `suspend` (Optional): Specifies whether the RayJob controller should create a RayCluster instance.
```

Comment:
Create a new section titled "Others" at the same level as "Submission configuration" and "Automatic resource cleanup. I know that you copied these lines from `rayjob_types.go`, but they're not very good. Use the following sentence instead:

"If `suspend` is true, KubeRay deletes both the RayCluster and the submitter. Note that Kueue also implements scheduling strategies by mutating this field. Avoid manually updating this field if you use Kueue to schedule RayJob.“

Reply:
Btw, you can also install Vale https://docs.ray.io/en/latest/ray-contribute/docs.html#how-to-use-vale to check whether you are following the Ray writing rules or not. 

A lot of old docs don't follow Vale, but we should do our best to follow Vale for new doc changes.